microhapdb/tests/test_frequency.py

```diff
@@ -95,3 +96,27 @@ def test_all_consistent():
         if len(numvars) > 1:
             inconsistent_markers.add(markerid)
     assert len(inconsistent_markers) == 0, sorted(inconsistent_markers)
+
+
+@pytest.mark.parametrize(
+    "marker,population,counts",
+    [
+        ("mh01CP-012", "EUR", [1052] * 4),
+        ("mh11HYP-29", "ASW", [122] * 3),
+        ("mh03USC-3qB", "CHS", [224] * 4),
+    ],
+)
+def test_counts_simple(marker, population, counts):
+    freq = microhapdb.frequencies
+    subset = freq[(freq.Marker == marker) & (freq.Population == population)]
+    assert subset.Count.to_list() == counts
+
+
+def test_counts_random():
+    freq = microhapdb.frequencies
+    pops = microhapdb.populations
+    for _ in range(5):
+        random_marker = choice(microhapdb.markers.Name)
+        random_population = choice(pops[pops.Source == "Byrska-Bishop2022"].ID.to_list())
+        subset = freq[(freq.Marker == random_marker) & (freq.Population == random_population)]
+        assert len(set(subset.Count)) == 1, subset
```

Comment:
For 5 random markers, test that the allele frequencies for a given population all have the same allele count.